doc/api/fs.md

```diff
@@ -619,6 +619,25 @@ On Linux, positional writes do not work when the file is opened in append mode.
 The kernel ignores the position argument and always appends the data to
 the end of the file.
 
+#### `filehandle.write(buffer, options)`
```

Comment:
```suggestion
#### `filehandle.write(buffer[, options])`
```

Reply:
With these changes, signatures will be overlapped:
[filehandle.write(buffer[, offset[, length[, position]]])](https://nodejs.org/api/fs.html#filehandlewritebuffer-offset-length-position)
[fs.write(fd, buffer[, offset[, length[, position]]], callback)](https://nodejs.org/api/fs.html#fswritefd-buffer-offset-length-position-callback)
[fs.writeSync(fd, buffer[, offset[, length[, position]]])](https://nodejs.org/api/fs.html#fswritesyncfd-buffer-offset-length-position)

Is it preferable? Behaviour with omitted arguments is indeed the same (and under the hood it is handled by "options" variant), but it would add ambiguity to docs.
Alternatively, all default values should be documented in current "positional" versions.

Reply:
Maybe make it the other way around: `options` should always be optional, offset can be required in the other signature (so `filehandle.write(buffer[, options])` and `filehandle.write(buffer, offset[, length[, position]])`)

Reply:
Hmm, done that. It still looks a bit weirder to me, but optionality of `options` and unambiguity outweights that ig.